itests/utilities/reward/reward.go

```diff
@@ -0,0 +1,85 @@
+		sumBalanceMinersGo, utl.GetHeightGo(suite), sumBalanceMinersScala, utl.GetHeightScala(suite))
+	// Get balances of dao and xtn buy-back accounts.
+	if addresses.DaoAccount != nil {
+		balanceDaoGo, balanceDaoScala = utl.GetAvailableBalanceInWaves(suite, addresses.DaoAccount.Address)
+	}
+	suite.T().Logf("Go: DAO balance: %d, Go current height:%d, Scala: DAO balance: %d, Scala current height: %d",
+		balanceDaoGo, utl.GetHeightGo(suite), balanceDaoScala, utl.GetHeightScala(suite))
+	if addresses.XtnBuyBackAccount != nil {
+		balanceXtnGo, balanceXtnScala = utl.GetAvailableBalanceInWaves(suite, addresses.XtnBuyBackAccount.Address)
+	}
+	suite.T().Logf("Go: XTN balance: %d, Go current height:%d, Scala: XTN balance: %d, Scala current height: %d",
+		balanceXtnGo, utl.GetHeightGo(suite), balanceXtnScala, utl.GetHeightScala(suite))
+	return utl.NewBalanceInWaves(sumBalanceMinersGo, sumBalanceMinersScala),
+		utl.NewBalanceInWaves(balanceDaoGo, balanceDaoScala), utl.NewBalanceInWaves(balanceXtnGo, balanceXtnScala)
+}
+
+func getDiffBalance(suite *f.BaseSuite, addressType string, currentBalance utl.BalanceInWaves, currentHeight uint64,
+	initBalance utl.BalanceInWaves, initHeight uint64) utl.BalanceInWaves {
+	diffBalanceGo := currentBalance.BalanceInWavesGo - initBalance.BalanceInWavesGo
+	diffBalanceScala := currentBalance.BalanceInWavesScala - initBalance.BalanceInWavesScala
+	suite.T().Logf("Go: Diff %s balance: %d on heights: %d - %d, Scala: Diff %s balance: %d, on heights: %d - %d",
+		addressType, diffBalanceGo, initHeight, currentHeight,
+		addressType, diffBalanceScala, initHeight, currentHeight)
+	return utl.NewBalanceInWaves(diffBalanceGo, diffBalanceScala)
+}
+
+func getAddressesDiffBalances(suite *f.BaseSuite, currentSumMinersBalance, currentDaoBalance, currentXtnBalance,
+	initSumMinersBalance, initDaoBalance, initXtnBalance utl.BalanceInWaves,
+	initHeight, currentHeight uint64) (utl.BalanceInWaves, utl.BalanceInWaves, utl.BalanceInWaves) {
+	// Diff sum miners balances.
+	diffMinersSumBalances := getDiffBalance(suite, "Miners", currentSumMinersBalance, currentHeight,
+		initSumMinersBalance, initHeight)
+	// Diff dao balances.
+	diffDao := getDiffBalance(suite, "DAO", currentDaoBalance, currentHeight, initDaoBalance, initHeight)
+	// Diff xtn.
+	diffXtn := getDiffBalance(suite, "XTN", currentXtnBalance, currentHeight, initXtnBalance, initHeight)
+	return diffMinersSumBalances, diffDao, diffXtn
+}
+
+func GetBlockRewardDistribution(suite *f.BaseSuite, addresses testdata.AddressesForDistribution) (
+	utl.RewardDiffBalancesInWaves, utl.RewardTerm) {
+	// Get balances in waves before block applied.
+	suite.T().Log("Balances before applied block: ")
+	initHeight := utl.GetHeight(suite)
+	initSumMinersBalance, initDaoBalance, initXtnBalance := getAddressesBalances(suite, addresses)
+	// Wait for 1 block.
+	utl.WaitForNewHeight(suite)
+	// Get balances after block applied.
+	suite.T().Log("Balances after applied block: ")
```

Comment:
Is it intended log record? I mean, it's unusual that it comes without additional info

Reply:
I think, yes, it is, because of debugging convenience.